src/main/java/org/prebid/server/auction/VideoStoredRequestProcessor.java

```diff
@@ -101,7 +101,8 @@ public VideoStoredRequestProcessor(ApplicationSettings applicationSettings, Vide
         if (StringUtils.isNotBlank(storedBidRequestId)) {
             storedRequestIds.add(storedBidRequestId);
         }
-        return applicationSettings.getVideoStoredData(storedRequestIds, podIds, timeoutFactory.create(defaultTimeout))
+        return applicationSettings.getVideoStoredData(null, storedRequestIds, podIds,
```

Comment:
Suggested alternative to passing `null` is overloaded method without this parameter.